pkg/registry/apis/dashboard/search.go

```diff
@@ -28,15 +27,13 @@ type SearchHandler struct {
 	log    log.Logger
 	client resource.ResourceIndexClient
 	tracer trace.Tracer
-	cfg    *setting.Cfg
 }
 
-func NewSearchHandler(cfg *setting.Cfg, client resource.ResourceIndexClient, tracer trace.Tracer) *SearchHandler {
```

Comment:
Avoid `cfg *setting.Cfg` in anything that should run multi-tenant.  The values are often a mix of single/multi-tenant so fail in cloud.

Specifically this was using it to construct the URL -- that value is actually different for each user